app/controllers/app/purchases_controller.rb

```diff
@@ -1,5 +1,15 @@
 class App::PurchasesController < App::BaseController
   def index
-    @purchases = current_user.purchases.order(created_at: :desc).includes(:item)
+    @purchases = current_user.purchases.includes(:item, :delivery_company).order(created_at: :desc)
+  end
+
+  def show
```

Comment:
https://www.notion.so/platanus/Gu-a-de-Estilo-67506d25082349d9b4128f25f1f81a69#f1e1397fa32642b1b66b6de8e451dbcd 

De acuerdo con la guía de estilos, en app mejor usar `set_purchase` para ser más explícitos 

<img width="748" alt="image" src="https://github.com/platanus/the-store1/assets/3026413/e4444ff8-a7c1-4720-9662-076b764040f7">